src/middlewared/middlewared/plugins/etc.py

```diff
@@ -29,7 +29,7 @@ class EtcGRP(enum.IntEnum):
     ROOT = 0
     SHADOW = 42
     NSLCD = 115
-    NUT = 126
+    NUT = 128
```

Comment:
Can this be read from `assets/account/builtin/linux/etc/group` instead of hardcode?